Source/Core/DolphinQt/Debugger/CodeDiffDialog.h

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QDialog>

```

Comment:
Done. Not familiar with C++: should it only be included in the .h and not in the cpp?

Reply:
Typically in Dolphin we do it both in the .h file and the .cpp file (assuming the header is actually needed in the .h file, otherwise it should not be included in the .h file). But I think DolphinQt in particular breaks this convention pretty often.

Reply:
Discussed in discord. Added to both.